seacrowd/sea_datasets/mysentence/mysentence.py

```diff
@@ -0,0 +1,149 @@
+# coding=utf-8
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
```

Comment:
Hi @Gyyz, could you please tidy up the `_CITATION` and add one more source (["Neural Sequence Labeling based Sentence Segmentation for Myanmar Language"](https://doi.org/10.1007/978-3-031-36886-8_24)) for the citation? Apparently [the dataset's github repo](https://github.com/ye-kyaw-thu/mySentence?tab=readme-ov-file#Publications) puts both sources as the publications.

Thanks for notifying us about this @sabilmakbar!! 👍